Makefile

```diff
@@ -0,0 +1,5 @@
+all:
```

Comment:
My Makefile knowledge is very limited (so this is more for my understanding) - do we need this all target or would the % one take care of this as well?

Reply:
Unfortunately it seems we need this target if we want "make" without an explicit target to work.